src/executables/util_service.c

```diff
@@ -233,12 +233,14 @@ static UTIL_SERVICE_OPTION_MAP_T us_Command_map[] = {
   {ACCESS_CONTROL, COMMAND_TYPE_ACL, MASK_SERVER | MASK_BROKER | MASK_GATEWAY},
   {RESET, COMMAND_TYPE_RESET, MASK_BROKER | MASK_GATEWAY},
   {INFO, COMMAND_TYPE_INFO, MASK_BROKER | MASK_GATEWAY},
-  {SC_COPYLOGDB, COMMAND_TYPE_COPYLOGDB, MASK_HEARTBEAT},
-  {SC_APPLYLOGDB, COMMAND_TYPE_APPLYLOGDB, MASK_HEARTBEAT},
   {GET_SHARID, COMMAND_TYPE_GETID, MASK_BROKER},
   {TEST, COMMAND_TYPE_TEST, MASK_BROKER},
+#if defined (ENABLE_UNUSED_FUNCTION)
```

Comment:
How about just commenting it out instead of ENABLE_UNUSED_FUNCTION with a short description?
Since 
1. it's simply semantically wrong. It's not a function.
2. When setting on the ENABLE_UNUSED_FUNCTION, the copylogdb and applylogdb are enabled. It seems weird.

What do you think?

Reply:
Agree ! 

`ENABLE_UNUSED_FUNCTION`
Throughout the current CUBRID source code, we are using the above macro for parts that are currently not being used (enums in system_parameter.h, functions, or parts of functions...). 

However, I agree with your suggestion that the macro should only be used for functions that are truly unused based on their intended purpose (semantic). I will follow your suggestion.